### DISCUSSION TITLE
Does The Program work for Linux 24.04.1 LTS

### DISCUSSION THREAD

**Discussion by Darkestangel69** [KevinBytesTheDust/CVAmp#302] [Q&A]
Hey everyone I got this popup today I was just wonder before I upgrade

Does The Program work for Linux 24.04.1 LTS?

**Comment by KevinBytesTheDust** [comment]
In theory it should, but we both experienced this `libharfbuzz` issue. If you can fix this on the Ubuntu side, it should run.
